README.adoc

```diff
@@ -163,13 +163,24 @@ configuration management process.
 === Prerequisites
 
 * Install repo and docker if it is not already done.
+* Install make and pkg-config if t is not already done.
 
-On Ubuntu, please run:
+On Ubuntu 20.04:
 
-  $ sudo apt-get install repo docker.io
+  * https://docs.docker.com/engine/install/ubuntu/[Install docker]
+  
+  * Install repo:
+  
+  $ sudo curl -o /usr/local/bin/repo https://storage.googleapis.com/git-repo-downloads/repo
+  $ sudo chmod +x /usr/local/bin/repo
+  $ sudo ln -s /usr/bin/python3 /usr/bin/python
```

Comment:
I prefer changing the repo script because this line will fail if python2 is installed.
`sed 's|/usr/bin/env python|/usr/bin/env python3|' -i /usr/local/bin/repo`